lib/greenhouse_io/api/client.rb

```diff
@@ -88,6 +92,17 @@ def get_from_harvest_api(url, options = {})
       end
     end
 
+    def post_to_harvest_api(url, body, on_behalf_of)
+      options = { :body => body.to_json, :basic_auth => basic_auth, :headers => {"On-Behalf-Of" => on_behalf_of.to_s} }
```

Comment:
Would `body.to_json` work outside of a Rails environment? I think it's only Rails which adds that instance method to `Object`. Just tried doing `Hash#to_json` from a fresh REPL session:

```
2.2.4 :002 > {:a => 5}.to_json
NoMethodError: undefined method `to_json' for {:a=>5}:Hash
	from (irb):2
	from /Users/gordon/.rvm/rubies/ruby-2.2.4/bin/irb:11:in `<main>'
```

Do you think it'd be better to use `JSON.dump`?

```
require 'json'
JSON.dump({:a => 5})
```